README.md

```diff
@@ -1,113 +1,283 @@
+let square5 = Square(topLeftCorner: (10.0, 10.0), lengthOfSide: 6.0)
+square5.lengthOfSide = 4.0  // NO! An error occurs
+```
 
+## Setting Computed Properties
 
-let powers = [
-    "Turn peanut butter into jelly",
-    "Sing like Michael Jackson",
-    "Teleport",
-    "Live on Mars"
-]
+In the previous examples, the computed property `area` was read-only; it could not be changed, only _read_. You can also create computed properties that can be changed, and can change other variables.
 
-let harry = Unicorn(name: "Harry", canFly: true, magicPowers: powers)
+It's easy to compute the area of a square based on the length of side. Of course, the reverse is also true: Given an area, it's easy to compute the length of a side, as well. The length of a side is simply the square root on the area.
 
+Here is an example of a `Square2` class which allows the area to be set directly:
 
+```swift
+class Square2 {
+    var topLeftCorner: (Double, Double)
+    var lengthOfSide: Double
+    var area: Double {
+        get {
+            return lengthOfSide * lengthOfSide
+        }
+        set {
+            lengthOfSide = sqrt(newValue)
+        }
+    }
 
-print(harry.description)
-/* prints "Harry is my name. I can fly!. Also, I have a lot of powers:
-Turn peanut butter into jelly
-Sing like Michael Jackson
-Teleport
-Live on Mars"
- */
+    init(topLeftCorner: (Double, Double), lengthOfSide: Double) {
+        self.topLeftCorner = topLeftCorner
+        self.lengthOfSide = lengthOfSide
+    }
+}
+```
 
+Notice how the computed property is defined. As before, it opens with a set of curly brackets. However, this time, there are two keywords, `get` and `set`. These each have a set of curly braces as well. Inside `get`'s curly braces is the previous definition of `area`: `return lengthOfSide * lengthOfSide`. However, the code inside `set`'s curly braces is new: `lengthOfSide = sqrt(newValue)`.
 
-print(harry.simpleDescription)
-// prints "Yo, my name is Harry"
+As you can guess, the code inside `get` is called when _reading_ the `area` property. The code inside `set` is called when _writing_, or assigning, the area property. It simply sets `lengthOfSide` to the square root of the new value (using the built-in `sqrt()` function). Notice that the value being assigned is not explicitly listed as a parameter; rather, the value is referred to as `newValue` inside the `set` code block.
+
+You can verify that this new computed property works as expected:
+
+```swift
+var square6 = Square2(topLeftCorner: (10.0, 10.0), lengthOfSide: 10.0)
+print("Area is 100.0? \(square6.area)")
+// Prints "Area is 100.0? 100.0"
+square6.area = 144.0
+print("Length of side is 12.0? \(square6.lengthOfSide)")
+// Prints "Length of side is 12.0? 12.0
 ```
 
+As before, if you want to change the value of a computed property, the object _must_ be a variable. If you had declared it as a constant, you would not be able to change `area`, _even though `area` is a variable computed property_.
```

Comment:
Similar to my above comment, but I will be making the change. Just making note for myself.